### PR TITLE
updated installation for Docker

### PR DESCRIPTION
**Pull request by Ananthavijay**
Fixes https://github.com/fossasia/badgeyay/issues/1673

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Updated instructions
@ParthS007 and @gabru-md review and merge.

Thanks,
Anantha Vijay.M

### CODE REVIEW

**Comment by Ananthavijay** [comment]
ok @ParthS007

**Comment by Ananthavijay** [comment]
@ParthS007 review it now

**Comment by Ananthavijay** [comment]
@ParthS007 sorry bro. I couldn't deploy it because my pc is crashed. I need to reinstall my os

File: api/docs/installation/docker.md [hunk 1]
```diff
@@ -1,20 +1,80 @@
...
+
+- Login to AWS and get to the Elastic Beanstalk service.
+- Create a new environment and select “Docker” as the preconfigured platform.
+- Upload a zip of your app files, which of course should include the Dockerfile. Create the zip from within the root directory of your app.
+
+Once you have successfully uploaded your files on the right size instance, Elastic Beanstalk will deploy your app and make it available.
+
+Cheers:
+
+## demo deployment :
```
Comment (xeon-zolt) [inline api/docs/installation/docker.md]: remove line 74 - 76

File: api/docs/installation/docker.md [hunk 2]
```diff
@@ -1,20 +1,80 @@
...
 
-* Get the latest version of docker. See the [offical site](https://docs.docker.com/engine/installation/) for installation info for your platform.
+### Prerequisites
 
-* Install the latest version of docker-compose. Windows and Mac users should have docker-compose by default as it is part of Docker toolbox. For Linux users, see the
-[official guide](https://docs.docker.com/compose/install/).
+Before you begin this, ensure the following is installed to your system:
 
-* Run `docker` and in terminal to see if they are properly installed.
+- Python 2.7 or 3.x,
```
Comment (xeon-zolt) [inline api/docs/installation/docker.md]: not required for docker

File: api/docs/installation/docker.md [hunk 3]
```diff
@@ -1,20 +1,80 @@
...
+- Docker, and
+- A git repository to store your project and track changes.
 
-* Clone the project and cd into it.
+## Create Dockerfile
 
-```bash
-git clone https://github.com/fossasia/badgeyay.git && cd badgeyay
-```
+Create a file named "Docker" with this content :
```
Comment (xeon-zolt) [inline api/docs/installation/docker.md]: replace docker with dockerfile

File: api/docs/installation/docker.md [hunk 4]
```diff
@@ -1,20 +1,80 @@
...
 
-* In the terminal window, run `sudo docker build -t badgeyay:latest .` to build badgeyay's docker image. This process can take some time.
+````
+FROM ubuntu:16.04
 
-* After successful build is done, run `sudo docker run -d -p 5000:5000 badgeyay` to start the server.
+MAINTAINER <your_name> "<your_mail> "
```
Comment (xeon-zolt) [inline api/docs/installation/docker.md]: Maintainer is deprecated

File: api/docs/installation/docker.md [hunk 5]
```diff
@@ -1,20 +1,80 @@
...
+Run this :
+````
+docker pull ubuntu:16.04
+docker run -it -v [full file path to your app files]:/app ubuntu:16.04 /bin/bash
+````
+## Fire up your Docker container
+Run this:
+``docker run -p 5000:5000 -v [file path to the project files on your computer]:/app fossasia/badgeyay``
+
+## Deploy on AWS Elastic Beanstalk
```
Comment (xeon-zolt) [inline api/docs/installation/docker.md]: this should be in separate file

**Comment by Ananthavijay** [comment]
guys @xeon-zolt @ParthS007 @gabru-md 
I will open a PR for this.

**Comment by Ananthavijay** [comment]
link : https://github.com/fossasia/badgeyay/pull/1687

**Pull request closed without merging**